src/themes/_common/style.scss

```diff
@@ -1,6 +1,11 @@
 .#{$csscommon}-full-width { width: 100%; }
 
-.#{$csscommon}-full-height { height: 100%; }
+.#{$csscommon}-full-height { 
+    height: 100%; 
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+}
```

Comment:
Hi @yoryi, this class is being used in a lot of different places, so I don't think it's a good idea to edit it. 😬 If we need to include more styles we should do it through a different class